src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
```

Comment:
The function `convertToObject` is not provided in the code. The task requires implementing this function, which takes a string with styles and converts it into an object with CSS declarations as keys and values.